eo-maven-plugin/pom.xml

```diff
@@ -169,13 +169,11 @@ SOFTWARE.
       <groupId>javax.json</groupId>
       <artifactId>javax.json-api</artifactId>
       <version>1.1.4</version>
-      <scope>runtime</scope>
     </dependency>
     <dependency>
       <groupId>org.glassfish</groupId>
       <artifactId>javax.json</artifactId>
       <version>1.1.4</version>
-      <scope>runtime</scope>
```

Comment:
@volodya-lombrozo are you sure? Usually `-api` stays in `compile`, while implementation stays in `runtime`.